src/Bridge/Doctrine/EventListener/PurgeHttpCacheListener.php

```diff
@@ -40,13 +40,15 @@ final class PurgeHttpCacheListener
     private $resourceClassResolver;
     private $propertyAccessor;
     private $tags = [];
+    private $xKeyPurger;
 
-    public function __construct(PurgerInterface $purger, IriConverterInterface $iriConverter, ResourceClassResolverInterface $resourceClassResolver, PropertyAccessorInterface $propertyAccessor = null)
+    public function __construct(PurgerInterface $purger, IriConverterInterface $iriConverter, ResourceClassResolverInterface $resourceClassResolver, PurgerInterface $xKeyPurger, PropertyAccessorInterface $propertyAccessor = null)
```

Comment:
I believe we should chain services & I'm not allowed to add another PurgerInterface like that ?

Reply:
yes you can but the argument must be at the last position and `nullable`